add HSA to v3.5 model update (#224)
The-Strategy-Unit/nhp_project_information: project_plan_and_summary/model_updates.qmd

We have recoded the data pipeline producing the population projection files, using PySpark on Databricks. This has meant that there are small differences in the population projection files, but these should have a very minimal impact on model results.

### Health Status Adjustment corrected

> There was an issue with the way in which provider activity rates were calculated for the [Health Status Adjustment](https://connect.strategyunitwm.nhs.uk/nhp/project_information/user_guide/setting_the_parameters.html#health-status-adjustment) element of the model. This has now been rectified. The impact of this on modelling results should be minimal.

Reviewer: Would be nice to say:

```suggestion
There was an issue with the way in which provider activity rates were calculated for the [Health Status Adjustment](https://connect.strategyunitwm.nhs.uk/nhp/project_information/user_guide/setting_the_parameters.html#health-status-adjustment) element of the model. This has now been rectified. The impact of this on modelling results is minimal.
```

Can we say that?
